core/slice.go

```diff
@@ -115,7 +116,10 @@ func (sl *Slice) Append(header *types.Header, domPendingHeader *types.Header, do
 	sl.phCachemu.Lock()
 	defer sl.phCachemu.Unlock()
 
-	log.Info("Starting slice append", "hash", header.Hash(), "number", header.NumberArray(), "location", header.Location(), "parent hash", header.ParentHash())
+	// Only print this if block is c_startingPrintLimit behind or less
+	if sl.CurrentInfo(header) {
+		log.Info("Starting slice append", "hash", header.Hash(), "number", header.NumberArray(), "location", header.Location(), "parent hash", header.ParentHash())
```

Comment:
Can we make this a debug print if not less than 10?

That way we can still trace higher granularity when desired and would achieve the same goal of reducing prints.

Reply:
good idea. will do that

Reply:
Done